We need to always run ldconfig first thing

Until ldconfig has ran, many of the other things we want to call as triggers
may not be able to find their libraries .... so this MUST run first


--- a/src/scripts.c
+++ b/src/scripts.c
@@ -96,6 +96,8 @@ void exec_post_update_script(bool reexec, bool block)
 
 static void update_triggers(bool block)
 {
+	system("/usr/bin/ldconfig");
+
 	if (strlen(POST_UPDATE) == 0) {
 		/* fall back to systemd if path prefix is not the rootfs
 		 * and the POST_UPDATE trigger wasn't specified */
